include/zephyr/net/wifi.h

```diff
@@ -484,6 +484,20 @@ static const char * const wifi_ps_param_config_err_code_tbl[] = {
 };
 /** @endcond */
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_WNM
+/** IEEE 802.11v BTM (BSS transition management) Query reasons.
+ * Refer to IEEE Std 802.11v - Table 7-43x-Transition and Transition Query reasons table.
```

Comment:
please add specific year, as sections can change and better to use 802.11-2020 and later versions

Reply:
Updated as comment